main.nf

```diff
@@ -102,54 +103,67 @@ def helpMessage() {
 // Show help message
 if (params.help) exit 0, helpMessage()
 
-// Handle deprecation
+// Default value for params
+
+// Set deprecated params to null
```

Comment:
Can you put all of this `params` initialisation into the config files? Unless it's a complex case requiring logic (eg. as we do with `params.genome` and the references), I usually try to put all params initialisation into the `base.config`. Keeps the main script a lot cleaner.